queue/src/fiskaltrust.Middleware.Localization.QueueIT/Constants/SignaturItemFactory.cs

```diff
@@ -0,0 +1,30 @@
+﻿using fiskaltrust.ifPOS.v1;
+using fiskaltrust.ifPOS.v1.it;
+using fiskaltrust.storage.V0;
+namespace fiskaltrust.Middleware.Localization.QueueIT.Constants
+{
+    public static class SignaturItemFactory
```

Comment:
Any specific reason for having a `SignatureItemFactory` and a `SignatureBuilder`? Can we merge them to one?

Reply:
Also a bit weird to have this stuff in "Constants", tbh :D